README.md

```diff
@@ -316,6 +319,45 @@ labels:
 - For each dataset, a `metadata.json` gets published listing all available files, for example: https://public-data.telemetry.mozilla.org/api/v1/tables/telemetry_derived/ssl_ratios/v1/files/metadata.json
 - The timestamp when the dataset was last updated is recorded in `last_updated`, e.g.: https://public-data.telemetry.mozilla.org/api/v1/tables/telemetry_derived/ssl_ratios/v1/last_updated
 
+Dataset Metadata
+---
+
+To provision a new BigQuery dataset for holding tables, you'll need to
+create a `dataset_metadata.yaml` which will cause the dataset to be
+automatically deployed.
```

Comment:
It's possibly worth mentioning that normal dataset deploy propagation to production takes 0.5-2 hours from when `generated-sql` is updated. There's currently no deploy notification to #dataops-alerts for this kind of deploy.